v0/destinations/af/transform.js

```diff
@@ -165,10 +165,8 @@ function processNonTrackEvents(message, eventName) {
 
 function processEventTypeTrack(message) {
   let isMultiSupport = true;
-  let isUnIdentifiedEvent = false;
   const evType = message.event && message.event.toLowerCase();
   let category = ConfigCategory.DEFAULT;
```

Comment:
@utsabc have we analysed the impact of any existing customers with removal of lowercase conversion ?

Reply:
We have analyzed the existing customers we are seeing none of them are sending. We have documented in the original card